third_party/move/move-compiler-v2/transactional-tests/tests/no-optimize-only/bug_13488_v1_err.exp

```diff
@@ -0,0 +1,98 @@
+comparison between v1 and v2 failed:
+- processed 5 tasks
+- 
+- task 0 'publish'. lines 1-44:
+- Error: error[E07006]: ambiguous usage of variable
```

Comment:
@vineethk @wrwg , errors below are only raised from V1. V2 does not generate the error because copy operation is implicitly generated in this PR.  Since no bytecode verification is raised, I think it is fine?